README.md

```diff
@@ -7,5 +7,11 @@
 
 Special thanks to [@monochromer](https://github.com/monochromer) for Eleventy transforms.
 
+## Development
+
+- Install dependencies: `npm install`
+- Start server for local development: `npm run start`
+- Run build for deployment: `npm run build`
```

Comment:
```suggestion
- Install dependencies: `npm install`
- Start local server: `npm start`
- Build for production: `npm run build`
- Lint code: `npm test`
```

Reply:
Yes, that's right. Corrected the text. Thank you